src/frontend/admin/package.json

```diff
@@ -28,6 +28,9 @@
     "@mui/x-date-pickers": "6.11.1",
     "@tanstack/react-query": "4.32.6",
     "@tanstack/react-query-devtools": "4.32.6",
+    "@uiw/react-markdown-preview": "4.1.16",
+    "@uiw/react-md-editor": "3.6.0",
+    "babel-plugin-transform-remove-imports": "^1.7.0",
```

Comment:
I feel link this should be in `devDependencies`, we don't need it after application build.